snowflake-account-grant-all/main.tf.json

```diff
@@ -0,0 +1,43 @@
+{
+  "//": "\n\tNOTE: Generated by scripts/snowflake_generate_grant_all.\n\t      Changes made directly to this file will be overwritten.\n\t\t\t\tMake improvements there so everyone can benefit.",
+  "variable": {
+    "per_privilege_grants": {
+      "type": "map(object({roles : list(string)}))",
+      "description": "A map of privileges to authorized roles and shares. Privileges must be UPPER case.\n  This allows you to authorize extra roles/shares for specific privileges.\n  The reason this module exists is that the provider only supports one grant resource per (database_name, schema_name, table_name, on_future, with_grant_option) tuple.\n\tFor example, if you used this module to grant an ALL privilege to a role you couldn't grant a subset of the ALL privs to another role.",
+      "default": {}
+    },
+    "roles": {
+      "type": "set(string)",
+      "description": "Grants privilege to these roles.",
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used